tests/conftest.py

```diff
@@ -1292,6 +1296,19 @@ def _disable_redact(request: pytest.FixtureRequest, mocker):
     return
 
 
+@pytest.fixture
+def airflow_root_path() -> Path:
+    import airflow
+
+    return Path(airflow.__path__[0]).parent
+
+
+@pytest.fixture
+def require_running_tests_from_sources() -> None:
+    if not (Path(__file__).parents[1] / "airflow" / "__init__.py").exists():
+        pytest.skip("This test requires tests to be run from sources. Skipping it")
```

Comment:
This should be replaced with a `skipif` marker instead. https://doc.pytest.org/en/latest/reference/reference.html#pytest-mark-skipif-ref

Reply:
Ah yeah. I was playing with it before, but yes it could be simplified this way.

Reply:
Fixed